recipes/r-asics/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = 'ASICS' %}
+{% set version = '1.0.1' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/Archive/{{ name }}/{{ name }}_{{ version }}.tar.gz
+  sha1: f0b6d0c44e26ee0350c9ec10e27d115c30307e63
+
+build:
+  number: 1
+  run_exports:
+    - {{ pin_subpackage("assembly-stats", max_pin="x.x.x") }}  
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: knitr, rmarkdown
+requirements:
+  build:
+    - r-base >3.6
+    - r-doparallel
+    - r-foreach
+    - r-ggplot2
+    - r-plyr
+    - r-quadprog
+    - r-zoo
+
+  run:
+    - r-base >3.6
+    - r-doparallel
+    - r-foreach
+    - r-ggplot2
+    - r-plyr
+    - r-quadprog
+    - r-zoo
+
+test:
+  commands:
+    - $R -e "library('ASICS')"
+
+about:
+  home: https://CRAN.R-project.org/package=ASICS
+  license: GPL (>= 2)
+  summary: 'With a set of pure metabolite spectra, ASICS quantifies  metabolites concentration
+    in a complex spectrum. The identification of  metabolites is performed by fitting
+    a mixture model to the spectra of the library with a sparse penalty. The method
+    and its statistical properties are  described in Tardivel et al. (2017) <doi:10.1007/s11306-017-1244-5>. '
+extra:
```

Comment:
It seems there is something wrong with the YAML syntax here. https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/112307/workflows/82778883-a387-4b8f-96c6-de3417b873ff/jobs/183143?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary says that there is no `linux-aarch64` entry and does not try to build on Linux ARM64